driver/csiplugin/controllerserver.go

```diff
@@ -575,23 +575,30 @@ func (cs *ScaleControllerServer) CreateVolume(ctx context.Context, req *csi.Crea
 	}
 
 	if isVolSource {
-		jobDetails := cs.Driver.volcopyjobstatusmap[scaleVol.VolName]
-		if jobDetails.jobStatus == VOLCOPY_JOB_RUNNING {
-			glog.Errorf("volume:[%v] -  volume cloning request in progress.", scaleVol.VolName)
-			return nil, status.Error(codes.Aborted, fmt.Sprintf("volume cloning request in progress for volume: %s", scaleVol.VolName))
-		} else if jobDetails.jobStatus == VOLCOPY_JOB_FAILED {
-			glog.Errorf("volume:[%v] -  volume cloning job had failed.", scaleVol.VolName)
-			return nil, status.Error(codes.Internal, fmt.Sprintf("volume cloning job had failed for volume: %s", scaleVol.VolName))
-		} else if jobDetails.jobStatus == VOLCOPY_JOB_COMPLETED {
-			glog.Infof("volume:[%v] -  volume cloning request has already completed successfully.", scaleVol.VolName)
-			return &csi.CreateVolumeResponse{
-				Volume: &csi.Volume{
-					VolumeId:      jobDetails.volID,
-					CapacityBytes: int64(scaleVol.VolSize),
-					VolumeContext: req.GetParameters(),
-					ContentSource: volSrc,
-				},
-			}, nil
+		jobDetails, found := cs.Driver.volcopyjobstatusmap.Load(scaleVol.VolName)
+		if found {
+			jobStatus := jobDetails.(VolCopyJobDetails).jobStatus
+			volID := jobDetails.(VolCopyJobDetails).volID
+			glog.Infof("volume: [%v] found in volcopyjobstatusmap with volID: [%v], jobStatus: [%v]", scaleVol.VolName, volID, jobStatus)
+			if jobStatus == VOLCOPY_JOB_RUNNING {
+				glog.Errorf("volume:[%v] -  volume cloning request in progress.", scaleVol.VolName)
+				return nil, status.Error(codes.Aborted, fmt.Sprintf("volume cloning request in progress for volume: %s", scaleVol.VolName))
+			} else if jobStatus == VOLCOPY_JOB_FAILED {
+				glog.Errorf("volume:[%v] -  volume cloning job had failed", scaleVol.VolName)
+				return nil, status.Error(codes.Internal, fmt.Sprintf("volume cloning job had failed for volume:[%v]", scaleVol.VolName))
+                	} else if jobStatus == VOLCOPY_JOB_COMPLETED {
+				glog.Infof("volume:[%v] -  volume cloning request has already completed successfully.", scaleVol.VolName)
```

Comment:
Add the the verbose level to logs - glog.Infof --> glog.V(5).Infof

Reply:
Done